task1/task1.js

```diff
@@ -0,0 +1,12 @@
+const reverseString = (str) => {
+    const words = str.split(/\W+/).reverse();
+    const chars = str.split(/\w+/);
+
+    let result = '';
+    for(let i = 0; i < words.length; i++) {
+        result = result + words[i].split('').reverse().join('') + chars[i];
+    }
+    return result
+}
+
+console.log(reverseString('Ab,c,de$!'));
```

Comment:
@elenagronskaya the implementation looks good, but could you make some changes to work with the input below as well?
`a,b$c` the result should be `c,b$a`